src/App.js

```diff
@@ -22,50 +23,140 @@ const generateSquares = () => {
     }
   }
 
+  // console.log(squares);
+  // squares[1][1].value = PLAYER_1;
+
   return squares;
-}
+};
 
 const App = () => {
+  // Wave 2
+  // You will need to create a method to change the square
+  //   When it is clicked on.
+  //   Then pass it into the squares as a callback
 
   // This starts state off as a 2D array of JS objects with
   // empty value and unique ids.
   const [squares, setSquares] = useState(generateSquares());
+  console.log(squares);
```

Comment:
When pushing code to be reviewed, we should eliminate any print statements or console logs.


```suggestion

```